apps/web/src/components/balances/AssetsTable/FiatChange.tsx

```diff
@@ -0,0 +1,43 @@
+import { Chip, SvgIcon, Typography } from '@mui/material'
+import { type Balance } from '@safe-global/store/gateway/AUTO_GENERATED/balances'
+import { formatPercentage } from '@safe-global/utils/utils/formatters'
+import ArrowDown from '@/public/images/balances/change-down.svg'
+import ArrowUp from '@/public/images/balances/change-up.svg'
+export const FiatChange = ({ balanceItem }: { balanceItem: Balance }) => {
```

Comment:
```suggestion

export const FiatChange = ({ balanceItem }: { balanceItem: Balance }) => {
```

Reply:
Changed it. We should put this into our lint rules.